### DISCUSSION TITLE
phpactor symfony

### DISCUSSION THREAD

**Discussion by eerison** [AstroNvim/astrocommunity#579] [Q&A]
Hello :)

Is there any example of how add others things like, symfony, tests, phpstan and so on in astrovim?

because here https://github.com/AstroNvim/astrocommunity/blob/main/lua/astrocommunity/pack/php/init.lua

I do not see any way to add others options 👀

**Comment by Uzaaft** [comment]
I'm sorry, but I don't understand your question. What are you trying to achieve?

**Comment by eerison** [comment]
Well I could make something works. 

It's missing make debug and tests integration works,  there is few docs how to make this works with php :/


But I guess it can be closed,

**Reply by Uzaaft** [reply]
If you phrased your question better, I could help you in figuring out how to implement it. :)

For the next time: https://github.com/selfteaching/How-To-Ask-Questions-The-Smart-Way/tree/master
